### DISCUSSION TITLE
支持rtl8821cu吗？

### DISCUSSION THREAD

**Discussion by l1uhong** [immortalwrt/immortalwrt#726] [General]
我在MT7628上运行OpenWRT-21.02，但是有一个问题，RTL8821CU不能正常工作，我使用dmesg命令检查，出现以下信息。
![image](https://user-images.githubusercontent.com/46449647/175239931-a4d93253-7b65-4ceb-b42f-9e8334676c22.png)

**Comment by 1715173329** [comment]
该模块仅在 x86 及 arm 上测试过可以工作。

https://github.com/Yuan-W/rtl8821cu/issues/2
有类似问题
